src/metax_api/api/rest/base/serializers/file_serializer.py

```diff
@@ -97,6 +97,11 @@ def is_valid(self, raise_exception=False):
             self.initial_data['parent_directory'] = self._get_id_from_related_object(
                 'parent_directory', self._get_parent_directory_relation)
 
+        if 'file_characteristics' in self.initial_data:
```

Comment:
```suggestion
        if 'file_characteristics' in self.initial_data and 'format_version' in self.initial_data['file_characteristics']:
```
because there are none else and would get rid of extra nested condition

Reply:
unless you disagree :D

Reply:
unless you disagree :)

Reply:
Good point!